vendor/modules.txt

```diff
@@ -877,6 +877,7 @@ github.com/prometheus/client_golang/prometheus/promauto
 github.com/prometheus/client_golang/prometheus/promhttp
 github.com/prometheus/client_golang/prometheus/push
 # github.com/prometheus/client_model v0.2.0
+## explicit
```

Comment:
this is odd, did you get this after running `make vendor-check` ?

Reply:
Correct.  The new reference is from reading the value of the prometheus counter in the tests, which seems good to have.